### PR TITLE
fix: :bug: Several fixes in the ACI integration

### PR DESCRIPTION
**Pull request by chadell**

### CODE REVIEW

**Comment by jdrew82** [comment]
What's the reasoning behind this? I didn't create the original ACI integration so I'm not sure why these strings were added to the DeviceRole slugs.

**Comment by chadell** [comment]
> What's the reasoning behind this? I didn't create the original ACI integration so I'm not sure why these strings were added to the DeviceRole slugs.

I'm using it, and I'm fixing some bugs, still not completed.
The idea, I guess, was to differentiate the role from ACI and other, but then this custom slug makes the name inconsistent and leads to errors.

File: nautobot_ssot/integrations/aci/diffsync/adapters/aci.py [hunk 1]
```diff
@@ -350,12 +351,25 @@ def load_devices(self):
...
+                        comments="",
+                    )
+                    self.add(_devicetype)
                 model = value["model"]
+
             new_device = self.device(
                 name=value["name"],
                 device_type=model,
-                device_role=value["role"],
+                device_role=value["model"],
```
Comment (chadell) [inline nautobot_ssot/integrations/aci/diffsync/adapters/aci.py:372]:
```suggestion
                device_role=value["role"],
```

**Comment by chadell** [comment]
@jdrew82 it is ready for review

File: nautobot_ssot/integrations/aci/diffsync/adapters/aci.py [hunk 2]
```diff
@@ -350,8 +351,21 @@ def load_devices(self):
...
+                    self.get("device_type", {"model": value["model"], "part_nbr": ""})
+                except ObjectNotFound:
+                    _devicetype = self.device_type(
+                        model=value["model"],
+                        manufacturer="Cisco",
+                        part_nbr="",
+                        u_height=1,
+                        comments="",
+                    )
+                    self.add(_devicetype)
```
Comment (Kircheneer) [inline nautobot_ssot/integrations/aci/diffsync/adapters/aci.py:366]: https://diffsync.readthedocs.io/en/latest/api/diffsync.html#diffsync.DiffSync.get_or_instantiate

```suggestion
                self.get_or_instantiate("device_type", ids={"model": value["model"], "part_nbr": ""}, attrs={"manufacturer": "Cisco", "u_height": 1, "comments": ""})
```

This should probably also work and be more concise

Comment (chadell) [reply nautobot_ssot/integrations/aci/diffsync/adapters/aci.py]: you are the expert ;)

**Pull request merged**